trace-utils/src/trace_utils.rs

```diff
@@ -526,6 +525,30 @@ pub fn enrich_span_with_mini_agent_metadata(
     }
 }
 
+pub fn enrich_span_with_google_cloud_function_metadata(
+    span: &mut pb::Span,
+    env_type: &EnvironmentType,
+) {
+    if let EnvironmentType::CloudFunction = env_type {
+        let function = span.meta.get("functionname");
```

Comment:
What do you think about passing the functionname in `self.config.app_name.clone()` as a parameter to the `enrich_span_with_google_cloud_function_metadata` method instead of fetching it from the span?

Reply:
Thanks, I added this